MainTel/MainTel-Troubleshooting/MTM2-Troubleshooting/MTM2_recovery.rst

```diff
@@ -0,0 +1,306 @@
+        :name: MTM2 Log message
+        :width: 600
+
+
+.. note::
+
+    Log Observations on the cause and comment on a Jira ticket (existing or created by you). Include unique activities that were occurring when it faulted (c.f. `OBS-416`_) as MTM2 is "not expected to fault".
+ 
+.. _`OBS-416` : https://rubinobs.atlassian.net/browse/OBS-416
+
+
+.. _MTM2-recovery-Procedure-Steps:
+
+Procedure Steps
+^^^^^^^^^^^^^^^
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+#. Restart control system.
+    .. warning::
+        This step must not be skipped. 
+        Restarting the control system acts as a catch-all for resetting issues. Failing to do so may also cause issues with telemetry.
+
+    #. **Connect** to the admin user on M2 cRIO controller via ssh using the username and password found in `1password`_ Main Telescope.
+
+        .. _`1password`: https://lsstit.1password.com/signin
+
+        .. code-block:: shell
+
+            ssh admin@m2-crio-controller01.cp.lsst.org
+
+    #. **Stop the control system** and wait 3 minutes using the command:
+    
+        .. code-block:: shell
+
+            /etc/init.d/nilvrt stop
+
+    #. **Start the control system** and wait 3 minutes using the command: 
+
+        .. code-block:: shell
+            
+            /etc/init.d/nilvrt start
+        
+        You may press enter to regain your shell prompt when you see the following "Welcome to LabVIEW Real-Time 18.0".
+
+        .. figure:: _static/MTM2_recovery_restart_control_system.png
+            :name: Restarting MTM2 control system
+            :width: 600
+
+    .. _MTM2-recovery-Reset-the-M2-interlock-signal: 
+
+#. `Reset the M2 interlock signal`_ in GIS main cabinet on the second floor, even if the state is "OK".
+
+    .. _`Reset the M2 interlock signal`: https://ts-m2gui.lsst.io/error-handling/error-handling.html#lsst-ts-m2gui-error-reset-m2-interlock-signal
+
+    .. Important::
+
+        Rarely, if ever, will the GIS show the M2 Actuator interlock in any state other than OK. This is even true when it is not OK.
+
+#. Use python EUI/GUI to change MTM2 to **Close Loop** status:
+
+    #. Open the **MTM2 EUI**. Follow instructions in `How to access the MTM2 EUI`_.
+
+    #. Establish **local control** with the EUI by hitting connect, then local (local may be greyed out after hitting connect, this is ok).
```

Comment:
Please follow standard of EUI button :guilabel:`Connect`  and :guilabel:`Local` 
https://obs-ops.lsst.io/project/observatory-operations-reST-reference.html#buttons-or-text-labels-in-interactive-programs